event_survey/views/event_event.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+
+  <record model="ir.ui.view" id="view_event_form">
+    <field name="name">event.event.survey</field>
+    <field name="model">event.event</field>
+    <field name="inherit_id" ref="event.view_event_form" />
+    <field name="arch" type="xml">
+      <xpath expr="//button[@name='toggle_active']" position="before">
+        <field name="survey_id"  invisible="True"/>
```

Comment:
bad indent

Reply:
I updated this,  but it's not showing us outdated, or I didn't understand where exactly the problem